decidim-dev/app/presenters/decidim/dev/official_author_presenter.rb

```diff
@@ -8,7 +8,7 @@ def name
       end
 
       def nickname
-        Decidim::UserBaseEntity.nicknamize(name)
+        ""
```

Comment:
I think this should actually be the value of `bin/rails runner "puts Decidim::UserBaseEntity.nicknamize(Decidim::Dev::OfficialAuthorPresenter.name, Decidim::Organization.first)"`

```suggestion
        "decidim_dev_official"
```